Osiris/Hacks/Multipoints.cpp

```diff
@@ -0,0 +1,170 @@
+#include "../Interfaces.h"
+#include "../Memory.h"
+
+#include "../SDK/Entity.h"
+#include "../SDK/Vector.h"
+#include "../SDK/matrix3x4.h"
+#include "../SDK/LocalPlayer.h"
+#include "../SDK/GlobalVars.h"
+#include "../SDK/ModelInfo.h"
+
+#include "../Hacks/Multipoints.h"
+
+namespace Multipoints
+{
+    bool retrieveAll(Entity* entity, float multiPointsExpansion, Vector(&multiPoints)[Multipoints::HITBOX_MAX][Multipoints::MULTIPOINTS_MAX])
+    {
+        if (localPlayer)
```

Comment:
you can do if(!localPlayer)
return false 
instead of an if(localPlayer)
//code 
same aplies to the rest

Reply:
You're right, I just wanted to have two 'return' only on each function. I will edit this soon.